app/helpers/lazy_render/lazyload_helper.rb

```diff
@@ -1,10 +1,11 @@
 module LazyRender
   module LazyloadHelper
-    def lazy_render(name, locals: {}, cache: false, callback: false)
+    def lazy_render(name, locals: {}, cache: false, callback: false, parallel: false)
```

Comment:
__[rubocop]__
* Line is too long. [85/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)